docs/integrations.md

```diff
@@ -0,0 +1,185 @@
+* Symbol - the name of your currency  (required)
+* Image - an image of your token  (not required)
+* Description - a brief description of your token (not required)
+* Recipient Address - automatically populates with your wallet address
+
+![token-tzen](/img/docs/integrations/thirdweb-token-tzen.png)
+
+4. Click **Deploy Now**. The **Deploy Status** page appears. It states that your wallet will prompt you to sign the transaction.
+
+![deploy-status](/img/docs/integrations/thirdweb-deploy-status.png)
+
+5. The MetaMask wallet displays a confirmation of the transaction. Click **Confirm**.
+
+![confirm-deploy](/img/docs/integrations/metamask-confirm-deploy-tzen.png)
+
+6. The MetaMask wallet displays a second confirmation of the transaction. Click **Confirm**.
+
+![confirm-second](/img/docs/integrations/metamask-confirm-second-tzen.png)
+
+7. The MetaMask wallet displays a signature notification. A signature is needed to add the transaction to the thirdweb Dashboard. Read the content fully before signing. Use the down-arrow to scroll to bottom the page.
+
+![signature-request](/img/docs/integrations/metamask-signature-request.png)
+
+8. Read and fully understand the content. Click **Sign**.
+
+![signature-second](/img/docs/integrations/metamask-signature-second.png)
+
+The **Deploy Status** notification appears displaying, **Adding to dashboard** message.
+
+![deploy-status](/img/docs/integrations/thirdweb-deploy-status-dashboard.png)
+
+When the process completes, the **Overview** page displays your token. Review your token details. You can now create additional tokens using the minting process. To mint, click **Tokens** (under Extensions) in the left-column menu.
+
+![tzen-details](/img/docs/integrations/thirdweb-tzen-details.png)
+
+9. Your contract token details are displayed. Since your token is newly deployed, the token’s **Total Supply** value is zero (0). Click **Mint**.
+
+![tzen=mint](/img/docs/integrations/thirdweb-tzen-mint.png)
+
+10. The **Mint additional tokens** page appears. In the **Additional Supply** field, enter an amount to mint additional tokens. In this example, 100 units are entered.
+
+![mint-token](/img/docs/integrations/thirdweb-mint-tokens.png)
+
+
+11. The MetaMask wallet displays a confirmation notification. Read the transaction details and click **Confirm**.
+
+![mint-confirm](/img/docs/integrations/metamask-mint-confirm.png)
+
+12. When the transaction completes, the **Tokens** page displays your token information. Review your token details. In this example, the Total Supply is the amount that is minted. 
+
+![tzen-verify](/img/docs/integrations/thirdweb-tzen-verify.png)
+
+## View Contracts
+
+A thirdweb Dashboard is a multi-purpose tool for deploying pre-built smart contracts, viewing the details of deployed smart contracts, and interacting with the contracts where you can modify its features, such as metadata, royalty fees, primary sales recipient, platform fees, and permission controls.

```

Comment:
is there a reason why we are capitalizing `Dashboard` here and throughout the doc? A dashboard seems like a general thing, instead of a specific branded product

Reply:
Also, could you split this up into two separate sentences?